src/ipvs/ip_vs_synproxy.c

```diff
@@ -296,13 +368,39 @@ static uint32_t syn_proxy_cookie_v4_init_sequence(struct rte_mbuf *mbuf,
             rte_atomic32_read(&g_minute_count), data);
 }
 
+__rte_unused static uint32_t
+syn_proxy_cookie_v6_init_sequence(struct rte_mbuf *mbuf,
+                                  struct dp_vs_synproxy_opt *opts)
+{
+    const struct ip6_hdr *ip6h = ip6_hdr(mbuf);
+    const struct tcphdr *th = tcp_hdr(mbuf);
+    int mssind;
+    const uint16_t mss = opts->mss_clamp;
+    uint32_t data;
+
+    /* XXX sort msstab[] by probability? Binary serarch? */
+    for (mssind = 0; mss > msstab[mssind + 1]; mssind++)
+        ;
+    opts->mss_clamp = msstab[mssind] + 1;
+
+    data = ((mssind & 0x0f) << DP_VS_SYNPROXY_MSS_BITS);
+    data |= opts->sack_ok << DP_VS_SYNPROXY_SACKOK_BIT;
+    data |= opts->tstamp_ok << DP_VS_SYNPROXY_TSOK_BIT;
+    data |= ((opts->snd_wscale & 0xf) << DP_VS_SYNPROXY_SND_WSCALE_BITS);
+
+    return secure_tcp_syn_cookie_v6(&ip6h->ip6_src, &ip6h->ip6_dst,
+            th->source, th->dest, ntohl(th->seq),
+            rte_atomic32_read(&g_minute_count), data);
+}
```

Comment:
syn_proxy_cookie_v6_init_sequence and yn_proxy_cookie_v4_init_sequence is almost the same ,whay not using like 
syn_proxy_cookie_v4_init_sequence(int af, XXXX,)

Reply:
Just to keep the same style with `secure_tcp_syn_cookie_xx` and `cookie_hash_xx`, which are more clear to use separate routines.